cypress/integration/GlossaryPopUps/GlossaryPopUps.js

```diff
@@ -0,0 +1,57 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here to Cypress from
```

Comment:
This code outside of the scope of a single test step will pass ALL errors - which makes this test file useless. Please put it in the step that requires ignoring a particular error